src/web/mjs/connectors/HiperCool.mjs

```diff
@@ -0,0 +1,73 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class HiperCool extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'hipercool';
+        super.label = 'Hiper Cool';
+        this.tags = [ 'manga', 'portugese', 'hentai' ];
+        this.url = 'https://hiper.cool/';
+    }
+
+    async _getMangas() {
+        let page = 1;
+        let morePages = true;
+
+        let request, data;
+
+        let mangas = [];
+        let uniq = new Set();
+
+        while ( morePages ) {
+            try {
+                request = new Request(new URL('/home/' + page, this.url), this.requestOptions);
+                data = await this.fetchDOM(request, '.news-list > li');
+                page++;
+
+                for (let manga of data) {
+                    if ( !uniq.has(manga.querySelector('a.news-thumb').pathname) ) {
+                        uniq.add(manga.querySelector('a.news-thumb').pathname);
+                        mangas.push(
+                            {
+                                id: manga.querySelector('a.news-thumb').pathname,
+                                title: manga.querySelector('div.title').innerText.trim().replace(/\s+\d{2}(?: Final)?$/, '')
+                            }
+                        );
+                    }
+                }
+            } catch(error) {
+                morePages = false;
+            }
+        }
+
+        return mangas;
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'ul.chapter-list li a.title');
+
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: "Chapter "+element.pathname.split('/').slice(-1)[0],
+                language: 'pt'
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.pages source');
+        return data.map(element => this.getRootRelativeOrAbsoluteLink(element, request.url));
```

Comment:
Page link should be absolute path instead of relative path

Reply:
It looks page links may still return the relative path, did you push all changes to the PR?

Reply:
right. don't know what happened. fixed now.